pkg/ddl/db_integration_test.go

```diff
@@ -1606,6 +1607,74 @@ func TestDefaultColumnWithRand(t *testing.T) {
 	tk.MustGetErrCode("CREATE TABLE t3 (c int, c1 int default a_function_not_supported_yet());", errno.ErrDefValGeneratedNamedFunctionIsNotAllowed)
 }
 
+func TestDefaultColumnWithReplace(t *testing.T) {
```

Comment:
Maybe we should move this to functional tests?

Reply:
There's an inconclusive inspection test here, so I didn't put it there.